src/quacc/schemas/vasp.py

```diff
@@ -293,7 +292,7 @@ def _bader_runner(path: Path | str) -> BaderSchema:
 
 
 def _chargemol_runner(
-    path: str, atomic_densities_path: str | None = None
+    path: Path | str, atomic_densities_path: str | None = None
```

Comment:
Just so I know for the future: should I be replacing all `Path | str` with `os.PathLike` or is that not wise?

Reply:
I believe the best option is to use  `Path | str` exclusively with Pathlib, and avoid combining `Pathlib` and `os.path`.